components/api_server/src/routes/auth.py

```diff
@@ -78,12 +79,12 @@ def get_credentials() -> tuple[str, str]:
 
 def get_ldap_config(username):
     """Get LDAP config from environment."""
-    ldap_search_filter_template = os.environ.get("LDAP_SEARCH_FILTER", "(|(uid=$username)(cn=$username))")
+    ldap_search_filter_template = getenv("LDAP_SEARCH_FILTER")
     return {
-        "ldap_root_dn": os.environ.get("LDAP_ROOT_DN", "dc=example,dc=org"),
-        "ldap_urls": os.environ.get("LDAP_URL", "ldap://localhost:389").split(","),
-        "ldap_lookup_user_dn": os.environ.get("LDAP_LOOKUP_USER_DN", "cn=admin,dc=example,dc=org"),
-        "ldap_lookup_user_pw": os.environ.get("LDAP_LOOKUP_USER_PASSWORD", "admin"),
+        "ldap_root_dn": getenv("LDAP_ROOT_DN"),
+        "ldap_urls": getenv("LDAP_URL").split(","),
```

Comment:
Too bad that the env file is not json.. Maybe it's worthwhile to implement json instead?

Reply:
I don't see what that would bring us? I think this is the only place where an environment variable is a little bit more complicated than a simple string.